lib/shopify_api/resources/base.rb

```diff
@@ -81,6 +81,7 @@ def persisted?
     end
 
     private
+
```

Comment:
Yeah, in Shopify core, you are supposed to have a new line after private. I saw it and fixed it here assuming gem should follow core's coding style guideline.